db/schema.rb

```diff
@@ -0,0 +1,56 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160218080947) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "drivers", id: false, force: :cascade do |t|
+    t.string   "email",            null: false
+    t.datetime "start",            null: false
+    t.string   "carplate",         null: false
```

Comment:
this is a bit of nitpicking sorry haha, but could create a migration to change from "carplate" to "car_plate"? Same for the referencing tables